app/javascript/packages/document-capture/components/document-capture.jsx

```diff
@@ -44,7 +33,13 @@ function DocumentCapture() {
     setFormValues(nextFormValues);
   }
 
-  const isFormEntriesError = submissionError && submissionError instanceof UploadFormEntriesError;
+  let initialActiveErrors;
```

Comment:
should we default this to `let initialActiveErrors = []` ?

Reply:
> should we default this to `let initialActiveErrors = []` ?

My thinking with this is to lean on `FormSteps` to provide the default:

https://github.com/18F/identity-idp/blob/4e2ff663418fe01ae2d08c17e9f64c8c3e2922bf/app/javascript/packages/document-capture/components/form-steps.jsx#L104

It doesn't make much difference here, but in other cases, it might be important to have this distinction from the rendering parent of: "We have no value to provide, define your own default behavior", represented by intentionally passing `undefined`.

Reply:
got it, works for me 👍